app/Helpers/functions.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+    function splitName($name)
+    {
+        $name = trim($name);
+        $nameArray = explode(" ", $name);
```

Comment:
If name of variable is camelCase (Camel case starts each new word in a complex variable name with an uppercase letter), you need set all variable same as.
$first_name => $firstName
$last_name => $lastName